sycl/include/sycl/multi_ptr.hpp

```diff
@@ -662,6 +662,7 @@ class multi_ptr<void, Space, DecorateAddress> {
 template <typename ElementType, access::address_space Space>
 class multi_ptr<ElementType, Space, access::decorated::legacy> {
 public:
+  using value_type = ElementType;
```

Comment:
This is not actually part of the spec, even after the changes from the mentioned spec PR. I have made a comment on it: https://github.com/KhronosGroup/SYCL-Docs/pull/432/files#r1251095640, but we should not add interfaces that aren't part of the spec.

Reply:
thanks @steffenlarsen. I agree with you on above. 
I would add it to spec for consistency with others.

Reply:
This is a good catch, the PR should have used `ElementType` instead of `value_type`, I've created a new PR to make this change to the spec (https://github.com/KhronosGroup/SYCL-Docs/pull/437), so I think here we can do the same and just use `ElementType` directly in `get_raw`.